pyproject.toml

```
@@ -1,14 +1,16 @@
 [project]
 name = "nemli"
-version = "0.1.0"
+version = "0.2"
```

@mattpsvreis 
Não fizemos nem a release da 0.1.0. De preferência pra ter 0 do patch tbm, facilita a vida do parser depois. Quando fizermos a release da 0.1.0 lançamos uma TAG no git. Por enquanto vamos trabalhando nessa versão até ter algo que chegue em produção.

```suggestion
version = "0.1.0"
```

---

Done ✅